src/lib/const.ts

```diff
@@ -4,7 +4,7 @@ export const version = require("../../package.json").version;
 export const minSchemaVersion = 0;
```

Comment:
Maybe I don't understand something, but how can the server support a client with a schema version < 29 when the return types (and also command arguments) have changed?

Reply:
that's a good point, thanks for calling that out!